packages/orama/src/types.ts

```diff
@@ -304,6 +304,7 @@ export interface IIndex<I extends OpaqueIndex = OpaqueIndex> {
     prop: string,
     id: string,
     value: SearchableValue,
+    schemaType: SearchableType,
```

Comment:
Since you add it here, I would add in `IIndexInsertOrRemoveHookFunction` as well and update relevant docs.

Reply:
Done